zktraffic/tests/test_stats_daemon.py

```diff
@@ -28,6 +30,20 @@
 from twitter.common.http.diagnostics import DiagnosticsEndpoints
 
 
+class FakeTimer(Timer):
```

Comment:
Aha. Correct me if I'm off: `reset` sets it up then prepares it for the next run, overriding `tick` just speeds up the process for testing.